streaming/index.js

```diff
@@ -520,12 +520,14 @@ const startWorker = (workerId) => {
 
   const onError = (err) => {
     log.error(err);
+    server.close();
+    process.exit(0);
```

Comment:
If there's an error, do we want to exit with code zero or something else?  Would this prevent supervisors from restarting errored processes (e.g. systemd in `Restart=on-failure` mode)?

Reply:
throng will trap process.exit and reboot the worker; this is why we were hitting an error loop before.